app/lib/frontend/static_files.dart

```diff
@@ -78,19 +82,25 @@ class StaticFile {
   );
 }
 
-final staticUrls = new StaticUrls(staticsCache.staticPath);
+final staticUrls = new StaticUrls();
```

Comment:
Suggestion: This is a global object that scans the filesystem in its constructor. That may be fine? Perhaps an initializer method or initializing on first use may be preferable?

Reply:
Fair point, I was already considering to make this more similar to the other `registerXYZ` initialization we have in other parts of the codebase, will do it in a follow-up.